# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/test/proxy_tunnel_socket_unittest.cpp
# Commit: f5353ea9f1df
# Full Hash: f5353ea9f1dfe15a65076bc7aa272d688ebdb8ca
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 2
# Description:
#   Bug 1569183: Add a proxy policy argument to NrSocketProxyConfig, and always set this config on the NrIceCtx. r=mjf
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45101
# ==============================================================================

diff -r ed1ae6a85255 -r f5353ea9f1df media/mtransport/test/proxy_tunnel_socket_unittest.cpp
--- a/media/mtransport/test/proxy_tunnel_socket_unittest.cpp	Tue Sep 17 18:17:13 2019 +0000
+++ b/media/mtransport/test/proxy_tunnel_socket_unittest.cpp	Tue Sep 17 18:17:14 2019 +0000
@@ -37,7 +37,8 @@
   void SetUp() override {
     nsCString alpn = NS_LITERAL_CSTRING("webrtc");
     std::shared_ptr<NrSocketProxyConfig> config;
-    config.reset(new NrSocketProxyConfig(0, alpn, net::LoadInfoArgs()));
+    config.reset(new NrSocketProxyConfig(0, alpn, net::LoadInfoArgs(),
+                                         NrSocketProxyConfig::kForceProxy));
     // config is never used but must be non-null
     mSProxy = new NrTcpSocket(config);
     int r = nr_socket_create_int((void*)mSProxy.get(), mSProxy->vtbl(),